task-definitions/survey-launcher.json

```diff
@@ -14,6 +14,20 @@
       {
         "name": "SURVEY_RUNNER_URL",
         "value": "${SURVEY_RUNNER_URL}"
+      },
+      {
+        "name": "JWT_ENCRYPTION_KEY_PATH",
+        "value": "${JWT_ENCRYPTION_KEY_PATH}"
+      },
+      {
+        "name": "JWT_SIGNING_KEY_PATH",
+        "value": "${JWT_SIGNING_KEY_PATH}"
+      }
+    ],
+    "mountPoints": [
+      {
+        "containerPath": "/jwt-keys",
```

Comment:
I think we should use `/secrets` rather than `jwt-keys` so it's consistent with what we're calling the mount. Also we may use it for other keys or secrets.